test-dts/ref.test-d.ts

```diff
@@ -168,6 +168,6 @@ expectType<Ref<number>>(toRef(obj, 'b'))
 // toRefs
 const objRefs = toRefs(obj)
 expectType<{
-  a: Ref<number>
+  a: number
```

Comment:
I think this is incorrect. `toRefs` wraps all values of `obj` in a ref, so the return type really should be `Ref<number>` here.

Reply:
I agree, I am not sure how to fix, so I closed this PR.